go/ephemeral/lib.go

```diff
@@ -319,16 +326,16 @@ func (e *EKLib) GetOrCreateLatestTeamEK(ctx context.Context, teamID keybase1.Tea
 		publishedMetadata = statement.CurrentTeamEkMetadata
 	}
 
-	_, err = teamEKBoxStorage.DeleteExpired(ctx, teamID, merkleRoot)
-	if err != nil {
-		return teamEK, err
-	}
 	teamEK, err = teamEKBoxStorage.Get(ctx, teamID, publishedMetadata.Generation)
 	if err != nil {
 		return teamEK, err
 	}
 	// Cache the latest generation
 	e.teamEKGenCache.Add(key, e.newCacheEntry(publishedMetadata.Generation))
+	_, err = teamEKBoxStorage.DeleteExpired(ctx, teamID, merkleRoot)
+	if err != nil {
+		return teamEK, err
+	}
```

Comment:
What's the importance of moving this around?

Reply:
just style